colander/__init__.py

```diff
@@ -205,6 +205,15 @@ def __str__(self):
         result of an execution of this exception's ``asdict`` method"""
         return pprint.pformat(self.asdict())
 
+
+class ExtraItemsError(Invalid):
+    def __init__(self, node, extras, msg=None):
+        if msg is None:
+            msg = _('Unrecognized items')
```

Comment:
You can fix the coverage failure by moving the `_('Unrecognize items')` into the function signature itself.  E.g.:

```python
class ExtraItemsError(Invalid):
    def __init__(self, node, extras, msg=_('Unrecognized items')):
        super(ExtraItemsError, self).__init__(node, msg)
        self.extras = extras
```